sdk/resourcemanager/azure-resourcemanager-monitor/src/test/java/com/azure/resourcemanager/monitor/AlertsTests.java

```diff
@@ -391,7 +388,9 @@ public void canCRUDMultipleResourceMetricAlerts() throws Exception {
             ma.refresh();
             Assertions.assertEquals(2, ma.scopes().size());
             Assertions.assertEquals(vm1.type(), ma.innerModel().targetResourceType());
-            Assertions.assertEquals(vm1.regionName(), ma.innerModel().targetResourceRegion());
+            if (!isPlaybackMode()) {
+                Assertions.assertEquals(vm1.regionName(), ma.innerModel().targetResourceRegion());
+            }
```

Comment:
What happens in playback? I'd assume region won't be redacted?

Reply:
The `targetResourceRegion` has been replace to `Sanitized` with latest TestProxy.